src/jdk.jartool/share/man/jar.md

```diff
@@ -213,6 +217,26 @@ operation modes:
 `--version`
 :   Prints the program version.
 
+## Integrity of a jar Archive
+As a jar archive is based on ZIP format, it is possible to create a jar archive using tools
```

Comment:
Nit - As a JAR file is based on ZIP format, it is possible to create a JAR file ...